src/web/data/userMapData.json

```diff
@@ -92,8 +99,9 @@
     "Belgium": [
       "Annuntia",
       "AP hogeschool Antwerpen",
+      "AP University College",
       "Artevelde University",
-      "Artevelde University College Ghent",
```

Comment:
Why the deletion? The official name of the college, based off Wiki, is `Artevelde University College Ghent`

Reply:
I removed it because when I looked Artevelde University on google it directed me to Artevelde University College Ghent, so it seemed redundent to have the same institute with 2 different names, but Ill change it and revert it to Artevelde University College Ghent